.readthedocs.yaml

```diff
@@ -12,6 +12,7 @@ build:
   commands:
     - python -m pip install --upgrade --no-cache-dir pip setuptools
     - python -m pip install --upgrade -r ./requirements.in -c ./requirements.txt
+    - sed -i '12 s/<!--\(.*\)-->/\1/' ./themes/ansible-community/templates/base.tmpl
```

Comment:
Why enable/disable as a sed? Can we not just commit the `meta` without the comment block?

Reply:
We don't want the `meta` in the final live version do we? I thought this way we can control when that gets inserted.

Reply:
Well sure, but we could just open an issue to remove it when the site finally goes live, no need to overcomplicate it ;)

Reply:
But we'd still want the preview site on RTD wouldn't we?

Reply:
clarifying the "possibly" - once the site is live is there any value in the preview? I see value in the PR previews, but the main branch should be on the live site anyway, right?

Reply:
Good point about forks and other deployments. It's probably also more reliable to remove the line as part of the "prod" build/deploy. If someone were to modify `base.tmpl` and the `meta` tag didn't stay on line 12 then the sed thing breaks.

As for the value of the readthedocs project after the final site is launched, I think we should keep it around for dev purposes. We might want to add a new band to the homepage or ditch nikola for pelican or whatever else. Having a test site makes a lot of sense.